src/pugixml.cpp

```diff
@@ -236,6 +236,15 @@ PUGI__NS_END
 PUGI__NS_BEGIN
 	template <typename T, typename D = void(*)(T*)> struct auto_deleter
 	{
+		// Non-copyable semantics
+#if __cplusplus >= 201103
```

Comment:
As discussed in the PR, C++11 deleted operators don't really give better error messages so they don't help - please remove this #if (as well as other #ifs of the same type below).